spec/features/make_a_booking_spec.rb

```diff
@@ -2,12 +2,11 @@
   scenario 'user can add a new booking' do
     connection = PG.connect(dbname: 'makersbnb_test')
     visit ('/')
+    find('button').click
+    visit ('/booking_form')
```

Comment:
Do you need this visit? Clicking a button should take you there?

Reply:
I think I don't need it actually, good catch Dean!